src/lib.rs

```diff
@@ -278,18 +278,21 @@ impl SpanExporter for StackDriverExporter {
   }
 }
 
-fn attribute_value_conversion(v: Value) -> AttributeValue {
-  use proto::google::devtools::cloudtrace::v2::attribute_value;
-  let new_value = match v {
-    Value::Bool(v) => attribute_value::Value::BoolValue(v),
-    Value::Bytes(v) => attribute_value::Value::StringValue(to_truncate(hex::encode(&v))),
-    Value::F64(v) => attribute_value::Value::StringValue(to_truncate(v.to_string())),
-    Value::I64(v) => attribute_value::Value::IntValue(v),
-    Value::String(v) => attribute_value::Value::StringValue(to_truncate(v)),
-    Value::U64(v) => attribute_value::Value::IntValue(v as i64),
-    Value::Array(_) => attribute_value::Value::StringValue(to_truncate(String::from(v))),
-  };
-  AttributeValue { value: Some(new_value) }
+impl From<Value> for AttributeValue {
+  fn from(v: Value) -> AttributeValue {
+    use proto::google::devtools::cloudtrace::v2::attribute_value;
+    AttributeValue {
+      value: Some(match v {
+        Value::Bool(v) => attribute_value::Value::BoolValue(v),
+        Value::Bytes(_) => attribute_value::Value::StringValue(to_truncate(v.into())),
```

Comment:
[This conversion operates on the assumption the underlying data is valid UTF-8](https://docs.rs/opentelemetry/0.8.0/src/opentelemetry/api/core.rs.html#179). That's not an assumption I want to go forward with. I do really like the rest of this PR! So if we could restore the hex encoding I'd be happy to merge it.

Reply:
Hm well seeing the `Bytes` type isn't present in the newer version of `opentelemetry` means I actually don't care about this line anymore as it's just going to get merge conflict resolved out of existence.